### PR TITLE
Typos in syndicate uplink descriptions

### LINKED ISSUE #27875: spelling mistake in codespeak uplink description
/datum/uplink_item/device_tools/codespeak_manual
	name = "Codespeak Manual"
desc = "Syndicate agents can be trained to use a series of codewords to  **_comvey_** complex information, which sounds like random concepts and drinks to anyone listening. This manual teaches you this Codespeak. You can also hit someone else with the manual in order to teach them. One use."

pretty sure it's "convey".

### PR DESCRIPTION
**Pull request by ShizCalev**
fixes #27875

### CODE REVIEW

**Review by Cyberboss — APPROVED** [review]

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -224,7 +224,7 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 
 /datum/uplink_item/dangerous/machinegun
 	name = "L6 Squad Automatic Weapon"
-	desc = "A fully-loaded Aussec Armoury belt-fed machine gun. \
+	desc = "A fully-loaded Aussec Armory belt-fed machine gun. \
```
Comment (Jordie0608) [inline code/modules/uplink/uplink_item.dm]: Inb4 dialect war starts again.

**Comment by ShizCalev** [comment]
There were 6 uses of the Americanized spelling of "armor" in the file versus three of the British "armour", just made the lesser used spelling match the majority for consistency reasons.

Doesn't matter much to me which way it goes, just trying to make things stay consistent.

**Comment by YakumoChen** [comment]
the room on the station is called "Armoury" if I recall? we should just change all the spelling to whatever we call it on station

**Comment by ShizCalev** [comment]
@yakumochan It's Armory on the station.

File: code/modules/uplink/uplink_item.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/uplink/uplink_item.dm]: Please undo this change and refer to https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#enforced-not-enforced for why

**Review by optimumtact — DISMISSED** [review]
Multiple changes of british -> american spelling for no good reason

**Comment by ShizCalev** [comment]
Gotcha. I'm in Vegas for the next 6 days, will do when I'm back.

**Comment by Cheridan** [comment]
:us: :us: :us: 
:us: :us: :us: 
:us: :us: :us:

**Comment by ShizCalev** [comment]
>no good reason

Honestly, I don't mind having both here and there, and I fully understand the reasoning behind it. However, with that said, having both present in a single interface describing items is just very awkward. If it was across multiple interfaces, it's whatevs, but it just seems to be weird to me to have the spelling of a single word switching forth between lines on one screen.
